src/screens/OrganizationPeople/addMember/AddMember.spec.tsx

```
@@ -28,6 +28,91 @@ vi.mock('components/NotificationToast/NotificationToast', () => ({
+  }: {
+    sorting: Array<{
+      testIdPrefix: string;
+      options: Array<{ value: string; label: string }>;
+      onChange: (value: string) => void;
+    }>;
+  }) => (
+    <div data-testid="page-header">
+      {sorting.map((sort, index) => (
+        <div key={index} data-testid={sort.testIdPrefix}>
+          {sort.options.map((opt) => (
+            <button key={opt.value} onClick={() => sort.onChange(opt.value)}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)